pkg/operator/ceph/cluster/osd/spec.go

```diff
@@ -210,11 +210,19 @@ func (c *Cluster) makeDeployment(osdProps osdProperties, osd OSDInfo, provisionC
 		// add the PVC size to the pod spec so that if the size changes the OSD will be restarted and pick up the change
 		envVars = append(envVars, v1.EnvVar{Name: "ROOK_OSD_PVC_SIZE", Value: osdProps.pvcSize})
 
-		// Append tuning flag if necessary
+		// Append slow tuning flag if necessary
```

Comment:
You need a spec test to validate the daemon CLI line which will check the newly added flags.

Reply:
Done.